src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -3091,7 +3092,9 @@ def test_api_video_patch_detail_token_user_estimated_duration_negative(self):
         jwt_token.payload["resource_id"] = str(video.id)
         jwt_token.payload["roles"] = [random.choice(["instructor", "administrator"])]
         jwt_token.payload["permissions"] = {"can_update": True}
-        data = {"estimated_duration": "-5400"}
+        # from -7 days to -1 second
+        estimated_duration = timedelta(seconds=random.randint(-604800, -1))
```

Comment:
why not just testing only -1? as you only test if days is < 0 I was curious to be sure that days where set to inf to 0 if estimated duration was less than a day

Reply:
I want to be sure that this test is working no matter the negative duration set.

Reply:
Ok can you add an extra test just with -1 to be sure as soon as it is negative it's thrown

Reply:
Done